Business Social Network/AppModules/Search/views/searchBox.html

```diff
@@ -1,12 +1,10 @@
-<html ng-app = "searchApp">
```

Comment:
@Swailem95 
create partials with a root div directory as it will be nested inside the main html tag